sdk/core/src/tasks/sboms/github/sync.rs

```diff
@@ -121,53 +124,77 @@ impl Service<Task> for SyncTask {
 impl SyncTask {
     /* PRIVATE */
 
-    /// Invokes the syft CLI against the cloned repository to generate an SBOM.
-    fn syft(
+    async fn ingest_sbom(
         &self,
-        source_path: &str,
-        full_name: &String,
-        commit_hash: String,
-    ) -> Result<String, Error> {
-        let output = match Command::new("syft")
-            .arg("--output")
-            .arg(CYCLONEDX_JSON_FORMAT)
-            .arg(source_path)
-            .output()
+        sbom: String,
+        url: String,
+        task: &mut Task,
+    ) -> Result<HashMap<String, String>, Error> {
+        let mut errors: HashMap<String, String> = HashMap::new();
+
+        let xref = Xref {
+            kind: XrefKind::Product,
+            map: HashMap::new(),
+        };
+
+        match self
+            .sboms
+            .ingest(sbom.as_str(), None, SbomProviderKind::GitHub, xref, task)
```

Comment:
I'm adding the HarborSyft kind to main and it should be there by tomorrow AM.

Reply:
Changed code to reflect this.